packages/react/src/modules/slack/components/SlackChannelCombobox/styles.css

```diff
@@ -0,0 +1,92 @@
+.rnf-trigger-container {
+  width: 256px;
+}
+
+.rnf-input-icon-container {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+}
+
+.rnf-input-with-icon {
+  padding-left: 30px;
+  height: 31px;
+  width: 100%;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+}
+.rnf-input-with-icon-error {
+  background-color: rgba(255, 255, 240, 1);
+  border: 1px solid rgba(247, 212, 102, 1);
+  cursor: not-allowed;
+}
+
+.rnf-input-with-icon-error::placeholder {
+  color: rgba(169, 88, 35, 1);
+}
+
+.rnf-input-icon {
+  position: absolute;
+  left: 10px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #ccc;
+}
+
+.rnf-input-icon-error {
+  color: rgba(247, 212, 102, 1);
+}
+
+.rnf-list-box {
+  margin-top: 4px;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  width: 286px;
+  margin-left: 30px;
+  height: 223px;
+  overflow-y: auto;
+  overflow-x: hidden;
+  box-shadow:
+    0px 1px 2px 0px rgba(0, 0, 0, 0.06),
+    0px 1px 3px 0px rgba(0, 0, 0, 0.1);
+}
+
+.rnf-channel-option {
+  display: flex;
+  padding: 8px 16px 8px 8px;
```

Comment:
Nit: we should try to use `rem` for any padding / margin / font declarations to help with a11y.